Thu Mar 06 18:45:25 CET 2025 : Apache Derby Network Server - 10.17.1.0 - (1913217) started and ready to accept connections on port 1527
Thu Mar 06 18:45:26 CET 2025 Thread[#24,DRDAConnThread_2,5,main] Cleanup action starting
java.sql.SQLException: Database 'sudoku' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:115)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
Caused by: ERROR XJ004: Database 'sudoku' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	... 17 more
============= begin nested exception, level (1) ===========
ERROR XJ004: Database 'sudoku' not found.
	at org.apache.derby.shared.common.error.StandardException.newException(StandardException.java:299)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.wrapArgsForTransportAcrossDRDA(SQLExceptionFactory.java:170)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:75)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(SQLExceptionFactory.java:141)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:225)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Util.java:220)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(EmbedConnection.java:3208)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(EmbedConnection.java:767)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(EmbedConnection.java:437)
	at org.apache.derby.iapi.jdbc.InternalDriver.getNewEmbedConnection(InternalDriver.java:629)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:295)
	at org.apache.derby.iapi.jdbc.InternalDriver.connect(InternalDriver.java:921)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(EmbeddedDriver.java:125)
	at org.apache.derby.impl.drda.Database.makeConnection(Database.java:253)
	at org.apache.derby.impl.drda.DRDAConnThread.getConnFromDatabaseName(DRDAConnThread.java:1474)
	at org.apache.derby.impl.drda.DRDAConnThread.verifyUserIdPassword(DRDAConnThread.java:1424)
	at org.apache.derby.impl.drda.DRDAConnThread.parseSECCHK(DRDAConnThread.java:3378)
	at org.apache.derby.impl.drda.DRDAConnThread.parseDRDAConnection(DRDAConnThread.java:1208)
	at org.apache.derby.impl.drda.DRDAConnThread.processCommands(DRDAConnThread.java:1008)
	at org.apache.derby.impl.drda.DRDAConnThread.run(DRDAConnThread.java:298)
============= end nested exception, level (1) ===========
Cleanup action completed
Thu Mar 06 18:45:26 CET 2025 Thread[#24,DRDAConnThread_2,5,main] (DATABASE = sudoku), (DRDAID = {1}), Database 'sudoku' not found.
----------------------------------------------------------------
Thu Mar 06 18:48:06 CET 2025:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.17.1.0 - (1913217): instance a816c00e-0195-6c8f-ea3a-ffff992e9aae 
on database directory C:\Users\piter\Desktop\Git_repos\Sudoku\DB\sudoku with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@33909752 
Loaded from file:/C:/Users/piter/Desktop/Git_repos/Sudoku_java/db-derby-10.17.1.0-bin/lib/derby.jar
java.vendor=Oracle Corporation
java.runtime.version=21.0.4+8-LTS-274
user.dir=C:\Users\piter\Desktop\Git_repos\Sudoku_java\db-derby-10.17.1.0-bin
os.name=Windows 11
os.arch=amd64
os.version=10.0
derby.system.home=C:\Users\piter\Desktop\Git_repos\Sudoku\DB
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Thu Mar 06 19:10:38 CET 2025: Shutting down Derby engine
----------------------------------------------------------------
Thu Mar 06 19:10:39 CET 2025:
Shutting down instance a816c00e-0195-6c8f-ea3a-ffff992e9aae on database directory C:\Users\piter\Desktop\Git_repos\Sudoku\DB\sudoku with class loader jdk.internal.loader.ClassLoaders$AppClassLoader@33909752 
----------------------------------------------------------------
